code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/integration/interfaces/IdentityTransformationHandler.java

```diff
@@ -0,0 +1,22 @@
+package io.cattle.platform.iaas.api.auth.integration.interfaces;
+
+import io.cattle.platform.api.auth.Identity;
+import io.cattle.platform.core.model.Account;
+
+import java.util.Set;
+
+public interface IdentityTransformationHandler extends Configurable {
+
+    Identity transform(Identity identity);
+
+    Identity untransform(Identity identity);
```

Comment:
untransform is never called. Can it be deleted from here and the implementations?